aspects/schema.go

```diff
@@ -264,13 +265,18 @@ func (v *mapSchema) Validate(raw []byte) error {
 	}
 
 	if missing {
-		return fmt.Errorf(`cannot find required combinations of keys`)
+		return &validationError{baseErr: fmt.Errorf(`cannot find required combinations of keys`)}
 	}
 
 	if v.entrySchemas != nil {
 		for key, val := range mapValue {
 			if validator, ok := v.entrySchemas[key]; ok {
 				if err := validator.Validate(val); err != nil {
+					var valErr *validationError
+					if errors.As(err, &valErr) {
+						valErr.path = append([]string{key}, valErr.path...)
+					}
+
```

Comment:
do we want the blank line here?

Reply:
Removed it since most similar places in this file don't have it